src/MonitorProfiler/Environment/EnvironmentHelper.h

```diff
@@ -18,11 +18,11 @@
 class EnvironmentHelper final
 {
 private:
-    static constexpr LPCWSTR DebugLoggerLevelEnvVar = _T("DotnetMonitorProfiler_DebugLogger_Level");
-    static constexpr LPCWSTR ProfilerVersionEnvVar = _T("DotnetMonitorProfiler_ProductVersion");
-    static constexpr LPCWSTR RuntimeInstanceEnvVar = _T("DotnetMonitorProfiler_RuntimeInstanceId");
-    static constexpr LPCWSTR SharedPathEnvVar = _T("DotnetMonitorProfiler_SharedPath");
-    static constexpr LPCWSTR StdErrLoggerLevelEnvVar = _T("DotnetMonitorProfiler_StdErrLogger_Level");
+    static constexpr LPCWSTR DebugLoggerLevelEnvVar = _T("DotnetMonitor_Profiler_DebugLogger_Level");
```

Comment:
I'm ok with this, but it looks a bit like configuration, even though it isn't. It might create confusion.